.github/workflows/tests.yaml

```diff
@@ -33,11 +33,15 @@ jobs:
         os:
           - ubuntu-latest
           - macos-13 # runs x64
+          - macos-latest # runs arm64
           - windows-latest
         python-version:
           - "3.9"
           - "3.10"
           - "3.11"
+        exclude:
+          - os: macos-13
+            python-version: "3.11"
```

Comment:
Apparently `libqasm0.6.6` breaks the `macos-13:python3.11` job, therefore decided to exclude it until fixed. (see [this msg ](https://qutech-sd.slack.com/archives/C05RRL1989L/p1716559203955159)for more info)